README.md

```diff
@@ -273,6 +274,12 @@ db-migrate supports the concept of environments. For example, you might have a d
     "driver": "sqlite3",
     "filename": ":memory:"
   },
+	
+  "int": {
```

Comment:
Ok i can do that. I personally thought this configuration example was very confusing. I was confused b/c you have some environment names as the same name as the driver.